app/views/passengers/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h2>Passenger Details</h2>
+
+<div><strong>Passenger: <%= @passenger.name %></strong>
+  <p><strong>Phone number: </strong><%= @passenger.phone_num %></p>
+  <p><strong>Total Amount Charged: $</strong><%= @passenger.total_charged %></p>
+  <strong>Passenger's Trips:</strong>
+  <ol>
+    <% @passenger.trips.each do |trip| %>
+      <li>Trip ID:<%= link_to trip.id, trip_path(trip.id) %>, driver: <%= link_to trip.driver.name, driver_path(trip.driver.id) %>, Date: <%= trip.date %>, rating: <%= trip.rating %>, cost: $<%= trip.money %></li>
+    <%end %>
```

Comment:
This can be done with a partial so you can re-use it any time you need a list of trips.